opm/test_util/summaryComparator.cpp

```diff
@@ -136,7 +138,7 @@ void SummaryComparator::getDeviation(size_t refIndex, size_t &checkIndex, Deviat
         return;
     }
     else if((*referenceVec)[refIndex]<(*checkVec)[checkIndex]){
-        double value = SummaryComparator::unitStep((*checkDataVec)[checkIndex-1]);
+        double value = SummaryComparator::unitStep((*checkDataVec)[checkIndex]);//Must be a little careful here. Flow writes out old value first, than changes value. Say there should be a change in production rate from A to B at timestep 300. Then the data of time step 300 is A and the next timestep will have value B. Must use the upper limit. 
```

Comment:
Try to limit the maximum length of the lines.  You don't have to press it into 80 characters, but 350 is a bit over the top.